### PR TITLE
Add CocoaPods spec and Swift 1.2 support

### LINKED ISSUE #3: Add podspec

### PR DESCRIPTION
**Pull request by a2**
Fixes #3 and #5.

### CODE REVIEW

File: Pistachio.podspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  s.homepage      = 'https://github.com/a2/Pistachio'
+  s.license       = 'MIT'
+  s.author        = { 'Felix Jendrusch' => 'felix@felixjendrusch.is',
+                      'Robert Böhnke' => 'robb@robb.is' }
+  s.source        = { :git => 'https://github.com/a2/Pistachio.git', :tag => s.version }
+  s.source_files  = 'Pistachio/*.{h,swift}'
+  s.dependency      'LlamaKit', '~> 0.6.0'
+  s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.9'
+end
```
Comment (felixvisee) [inline Pistachio.podspec:14]: Doesn't really match the original repository 😥

**Comment by felixvisee** [comment]
I'm thinking about doing some changes that remove unnecessary complexity before moving on to Swift 1.2 (unless it goes out of beta before that). Keeping this open for now, thanks.

**Comment by a2** [comment]
If I made a branch with just an updated .podspec without the Swift 1.2 changes, would you merge that?

**Comment by felixvisee** [comment]
Yo, that sounds good!

**Comment by felixvisee** [comment]
See #10.

**Pull request closed without merging**